models/product.model.lkml

```diff
@@ -216,6 +218,12 @@ explore: rpt_active_user_base {
   group_label: "[New] Active Users"
 }
 
+explore: fct_inapp_trial_requests  {
```

Comment:
we prob. need to reason out (or document) the diff between `fct_in_**product**_trial_requests` and `fct_**inapp**_trial_requests`

Reply:
I was thinking about this too. "In-app" is also misleading in both cases. Perhaps start with the current one renaming it from `fct_inapp_trial_requests` to `fct_onprem_trial_requests_history`?

Reply:
Might also rename the `fct_in_product_trial_requests` to `fct_inapp_trial_requests`, so that we don't introduce another synonym to in-app (similar to on-prem where we also have self-hosted).

Reply:
I was actually thinking of the following changes:
- Use a new prefix instead of `fct` (i.e. `sync`) to indicate that this table is intended for syncing to remote systems
- Use on-prem instead of in-app because in reality it's both in-app and website for on-prem trials (didnt' know that when the `fct_in_product_trial_requests` was created)
- Add the proper granularity (latest per email).

This would result in renaming `fct_in_product_trial_requests` to `sync_onprem_trial_request_latest_per_email`. If that's OK, I'll create a separate ticket to handle such changes, as it also affects hightouch configuration.

Reply:
Sounds great!